pyscriptjs/src/utils.ts

```diff
@@ -50,21 +50,21 @@ export function handleFetchError(e: Error, singleFile: string) {
     // fragile. We need a better solution.
     let errorContent: string;
     if (e.message.includes('Failed to fetch')) {
-        errorContent = `<p>PyScript: Access to local files
+        errorContent = `PyScript: Access to local files
         (using "Paths:" in &lt;py-config&gt;)
         is not available when directly opening a HTML file;
         you must use a webserver to serve the additional files.
         See <a style="text-decoration: underline;" href="https://github.com/pyscript/pyscript/issues/257#issuecomment-1119595062">this reference</a>
-        on starting a simple webserver with Python.</p>`;
+        on starting a simple webserver with Python.`;
     } else if (e.message.includes('404')) {
         errorContent =
-            `<p>PyScript: Loading from file <u>` +
+            `PyScript: Loading from file <u>` +
             singleFile +
-            `</u> failed with error 404 (File not Found). Are your filename and path are correct?</p>`;
+            `</u> failed with error 404 (File not Found). Are your filename and path are correct?`;
     } else {
-        errorContent = `<p>PyScript encountered an error while loading from file: ${e.message} </p>`;
+        errorContent = `PyScript encountered an error while loading from file: ${e.message}`;
     }
-    throw new UserError(errorContent, "html");
+    throw new UserError(UserError.ErrorCode.FETCH_ERROR, errorContent, "html");
 }
```

Comment:
[follow-up]
this seems to be related to the comment which I made above about `FetchUserError` & co.
I think that we can improve this messy logic: instead of having a "catch-all" function which tries to interpret the error *after the fact*, we could probably be proactive: every call to `fetchRobust(userError=True)` (pseudocode) should also indicate an user-friendly explanation. 
 
For example, if we fetch something from the `[[fetch]]` config, saying `Check your filename` might be useful. But if we fail to fetch some internal file (e.g. the pyodide stdlib), then saying `Check your filename` is just confusing to the user. But again, this sounds like a topic for a separate discussion